src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -937,12 +925,6 @@ const getTogglerTheme = ({
     background: ${buttonTheme.background};
     border: 1px solid ${buttonTheme.border};
     color: ${buttonTheme.color};
-
-    &:hover {
```

Comment:
@doralchan is this intentional?

Reply:
Yeah - its kinda weird to have that appear white on hover

Reply:
Ok gotcha. Just checking 👍